src/main/app/src/app/zaken/zaken.service.ts

```diff
@@ -104,8 +104,8 @@ export class ZakenService {
     return this.zacHttpClient.PUT("/rest/zaken/lijst/verdelen", body);
   }
 
-  vrijgevenVanuitLijst(body: PutBody<"/rest/zaken/lijst/vrijgeven">) {
-    return this.zacHttpClient.PUT("/rest/zaken/lijst/vrijgeven", body);
+  vrijgevenVanuitLijst() {
```

Comment:
I don't get how the required body information is injected here. I see it is passed into the `mutation`, but from there I do not see a way that information ends up in the call...

Reply:
This magic doesn't make it easier to understand.